app/(app)/orgs/[orgId]/tools/scan-to-task/_components/s2t-history-pagination.ts

```
@@ -0,0 +1,184 @@
+      const response = await fetch(buildUrl(cursor));
+      const data = (await response.json()) as HistoryPage;
+      if (requestId !== requestIdRef.current) return;
+      applyHistoryPage(data, append);
+    } catch {
+      if (requestId !== requestIdRef.current) return;
+      if (append) return;
+      setResults([]);
+      setDuplicateCandidatesById({});
+      setNextCursor(null);
+    } finally {
+      if (requestId !== requestIdRef.current) return;
```

Biome (2.5.5)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)